webpack/assets/javascripts/react_app/components/common/forms/OrderableSelect/components/OrderableToken.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Token } from 'react-bootstrap-typeahead';
+
+import { orderable } from '../helpers';
+
+const orderConfig = {
+  type: 'multiValue',
+  getItem: props => ({ value: props.data.value }),
+  getIndex: props => props.data.index,
+  getMoveFnc: props => props.moveDraggedOption,
+};
+
+const OrderableToken = ({
+  isDragging,
+  moveDraggedOption,
+  data,
+  disabled,
+  onRemove,
+  tabIndex,
+  labelKey,
+}) => (
+  <Token disabled={disabled} onRemove={onRemove} tabIndex={tabIndex}>
```

Comment:
I think we can get Token from patternfly-react, maybe something like:

```js
import { TypeAheadSelect } from 'patternfly-react';
// snip
<TypeAheadSelect.Token>
```

Reply:
We can't because patternfly-react is not exporting it :(